src/display.cpp

```diff
@@ -1075,7 +1075,8 @@ void display::get_terrain_images(const map_location& loc, const std::string& tim
 {
 	terrain_image_vector_.clear();
 
-	image::light_string lt;
+	static std::vector<image::light_adjust> lighting;
```

Comment:
why is this static?

Reply:
Was following the example of the terrain image vector which was a class member instead of being reallocated every call. Should probably just remove the static.

Reply:
I'll add to here once you've pushed updates to master.

Reply:
Tracking #10158